node/src/command.rs

```diff
@@ -55,28 +103,6 @@ impl<T: sc_service::ChainSpec + 'static> IdentifyChain for T {
 	}
 }
 
-fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
```

Comment:
I broke this function down into the `create_chain_spec` and `load_chain_spec_from_json_file` functions of `ChainIdentity`. This simplified this function so that I could directly move the simplified content to [where it is used](https://github.com/pendulum-chain/pendulum/pull/302/files#diff-8ce03a5a2b79f3021ff2d6322d13448db8cfd5fb9267517ab4b96d0d299ed45fR137).